policy/Azure-Proactive-Resiliency-Library-v2/azapi/compute/microsoft_compute_virtualMachineScaleSet_disable_force_strictly_even_balance.rego

```diff
@@ -0,0 +1,15 @@
+package Azure_Proactive_Resiliency_Library_v2
+
+import rego.v1
+
+valid_azapi_virtual_machine_scaleset_disable_force_strictly_even_zone_balancing(resource) if {
+    resource.values.body.properties.zoneBalance == false
```

Comment:
According to [the ARG Query](https://azure.github.io/Azure-Proactive-Resiliency-Library-v2/azure-resources/Compute/virtualMachineScaleSets/#disable-force-strictly-even-balance-across-zones-to-avoid-scale-in-and-out-fail-attempts) it should be considered as violation when `properties.zoneBalance == true`, so what if `properties.zoneBalance` hasn't been set?